pkg/relayer/miner/gen/gen_fixtures.go

```diff
@@ -108,7 +108,7 @@ func main() {
 	}
 
 	// Write the output buffer to the file at flagOut path.
-	if err := os.WriteFile(flagOut, outputBuffer.Bytes(), 0644); err != nil {
+	if err := os.WriteFile(flagOut, outputBuffer.Bytes(), 0o644); err != nil {
```

Comment:
I find the explicit octal representation less intuitive and idiomatic. Which rule enforces this?

Reply:
That was my linter again will revert

Reply:
I think go prefers octals here

Reply:
gofumpt does this - it is actually a go standard it seems.